backend/LexBoxApi/GraphQL/LexMutations.cs

```diff
@@ -96,4 +96,30 @@ await dbContext.ProjectUsers.Where(pu => pu.ProjectId == input.ProjectId && pu.U
             .ExecuteDeleteAsync();
         return dbContext.Projects.Where(p => p.Id == input.ProjectId).AsExecutable();
     }
+
+    [Error<NotFoundException>]
+    [Error<DbError>]
+    //[Error<RequiredException>]
+    [UseMutationConvention]
+    public async Task<User> ChangeUserAccountData(ChangeUserAccountDataInput input, LexBoxDbContext dbContext)
+    {
+        var user = await dbContext.Users.FindAsync(input.UserId); //find based on userId
```

Comment:
There are a few places where the code itself is pretty much as readable as your added comment. If that's the case (like it is here), then you can remove the comment.